mslib/mswms/wms.py

```diff
@@ -274,6 +274,7 @@ def generate_gallery(self, create=False, clear=False, generate_code=False, sphin
                                 init_time = plot_driver.get_init_times()[-1]
                                 valid_time = plot_driver.get_valid_times(plot_object.required_datafields[0][1],
                                                                          file_type, init_time)[-1]
+                                style = plot_object.styles[0][0] if plot_object.styles else None
```

Comment:
Isnt't this one [0] to much? styles is a list of strings...?

Reply:
styles is a list of 2-tuples containing strings.